app/src/main/res/values/strings.xml

```diff
@@ -88,6 +87,9 @@
     <string name="title_skip_login">Skip Login</string>
     <string name="title_edit_detail">Edit</string>
     <string name="action_logout">Logout</string>
+    <string name="msg_not_booking">You still not booking yet</string>
+    <string name="title_na">N/A</string>
+    <string name="msg_book_to_edit">You can book again to edit your last booking</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.msg_book_to_edit` appears to be unused
 Unused resources make applications larger and slow down builds.